src/styles/index.scss

```diff
@@ -1,3 +1,144 @@
-body {
+.page {
+  scroll-behavior: smooth;
+  background: #fff;
+  font-family: Roboto, sans-serif;
+
+  &_body {
+    margin: 0;
+  }
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+  padding-left: 16px;
+  padding-right: 16px;
+  box-sizing: border-box;
+}
+
+img {
+  display: block;
+  margin: 32px auto 40px;
+  padding-right: 2px;
+  padding-left: 2px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name-product {
   margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  width: 100%;
+
+  color: #060b35;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+  margin-top: 4px;
+  margin-bottom: 16px;
+}
+
+.card__grade {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  gap: 4px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  align-items: center;
+  gap: 4px;
+
+  &--1 :first-child,
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(n + 1) {
+    background-image: url(../images/star-active.svg);
+    color: #ffde6a;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.card__reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: right;
+}
+
+.card__price {
+  width: 100%;
+  padding-top: 24px;
+  padding-bottom: 16px;
+
+  display: flex;
+  justify-content: space-between;
+}
+
+.price {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+  color: #616070;
+}
+
+.number {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  text-align: right;
+  color: #060b35;
+}
+
+a {
+  display: inline-flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 16px;
+  box-sizing: border-box;
+}
+
+.card__button {
+  width: 100%;
+  border-radius: 5px;
+  background-color: #00acdc;
+  color: #fff;
+  text-align: center;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+
+  &:hover {
+    width: 100%;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+    background-color: #fff;
+    color: #00acdc;
```

Comment:
Consider using variables for repeating colors

Reply:
I'll try!